Astropy dev test failure for HGS->HCRS.

Currently with the latest version of astropy master. 

```python
def test_hgs_hcrs():
        # This test checks the HGS->HCRS transformation by transforming from HGS to
        # HeliocentricTrueEcliptic (HTE).  It will fail if there are errors in Astropy's
        # HCRS->ICRS or ICRS->HTE transformations.
    
        # Use published HGS coordinates in the Astronomical Almanac (2013), pages C6-C7
        obstime = Time('2013-01-28')
        earth_hgs = SkyCoord(0*u.deg, -5.73*u.deg, 0.9848139*u.AU, frame=HeliographicStonyhurst,
                             obstime=obstime)
    
        # Transform to HTE at observation-time equinox
        earth_hte = earth_hgs.transform_to(HeliocentricTrueEcliptic(equinox=obstime))
    
        # Validate against published values from the Astronomical Almanac (2013), page C6 per page E2
        # The dominant source of inaccuracy is the limited precision of the published B0 used above
>       assert quantity_allclose(earth_hte.lon, Angle('308d13m30.51s') - 180*u.deg, atol=5*u.arcsec)
E       assert False
E        +  where False = quantity_allclose(<Longitude 128.23007468 deg>, (<Angle 308.22514167 deg> - (180 * Unit("deg"))), atol=(5 * Unit("arcsec")))
E        +    where <Longitude 128.23007468 deg> = <SkyCoord (HeliocentricTrueEcliptic: equinox=2013-01-28 00:00:00.000, obstime=2013-01-28 00:00:00.000): (lon, lat, distance) in (deg, deg, AU)\n    (128.23007468, 0.00023427, 0.9848139)>.lon
E        +    and   <Angle 308.22514167 deg> = Angle('308d13m30.51s')
E        +    and   Unit("deg") = u.deg
E        +    and   Unit("arcsec") = u.arcsec
```

Need to figure out what's changed and if it's ok.

--- a/changelog/3075.bugfix.rst
+++ b/changelog/3075.bugfix.rst
@@ -1,0 +1,1 @@
+Fix HGS <> HCRS test due to Ecliptic frame changes in astropy 3.2

--- a/sunpy/coordinates/tests/test_transformations.py
+++ b/sunpy/coordinates/tests/test_transformations.py
@@ -3,8 +3,13 @@ import pytest
 
 import astropy.units as u
 from astropy.tests.helper import quantity_allclose, assert_quantity_allclose
-from astropy.coordinates import (SkyCoord, get_body_barycentric, HeliocentricTrueEcliptic, Angle,
+from astropy.coordinates import (SkyCoord, get_body_barycentric, Angle,
                                  ConvertError)
+try:
+    from astropy.coordinates import HeliocentricMeanEcliptic
+except ImportError:
+    from astropy.coordinates import HeliocentricTrueEcliptic as HeliocentricMeanEcliptic
+
 from astropy.time import Time
 
 from sunpy.coordinates import (Helioprojective, HeliographicStonyhurst,
@@ -136,13 +141,13 @@ def test_hgs_hcrs():
                          obstime=obstime)
 
     # Transform to HTE at observation-time equinox
-    earth_hte = earth_hgs.transform_to(HeliocentricTrueEcliptic(equinox=obstime))
+    earth_hme = earth_hgs.transform_to(HeliocentricMeanEcliptic(equinox=obstime))
 
     # Validate against published values from the Astronomical Almanac (2013), page C6 per page E2
     # The dominant source of inaccuracy is the limited precision of the published B0 used above
-    assert quantity_allclose(earth_hte.lon, Angle('308d13m30.51s') - 180*u.deg, atol=5*u.arcsec)
-    assert quantity_allclose(earth_hte.lat, -Angle('-0.27s'), atol=10*u.arcsec)
-    assert quantity_allclose(earth_hte.distance, 0.9848139*u.AU, atol=5e-7*u.AU)
+    assert quantity_allclose(earth_hme.lon, Angle('308d13m30.51s') - 180*u.deg, atol=5*u.arcsec)
+    assert quantity_allclose(earth_hme.lat, -Angle('-0.27s'), atol=10*u.arcsec)
+    assert quantity_allclose(earth_hme.distance, 0.9848139*u.AU, atol=5e-7*u.AU)
 
 
 def test_hgs_hgc_roundtrip():

